frontend/packages/dev-console/src/components/add/AddCardsLoader.tsx

```diff
@@ -0,0 +1,53 @@
+import * as React from 'react';
+import { connect } from 'react-redux';
+import { LoadingBox } from '@console/internal/components/utils';
+import { RootState } from '@console/internal/redux';
+import {
+  useResolvedExtensions,
+  AddActionGroup,
+  isAddActionGroup,
+} from '@console/dynamic-plugin-sdk';
+import AddCardSection from './AddCardSection';
+import { useAddActionExtensions } from '../../utils/useAddActionExtensions';
+
+interface StateProps {
+  activeNamespace: string;
+}
+
+interface OwnProps {
+  title: string;
+  hintBlock?: React.ReactNode;
+}
+
+type AddCardsLoaderProps = OwnProps & StateProps;
+
+export const AddCardsLoader: React.FC<AddCardsLoaderProps> = ({
```

Comment:
Seems like this component is exactly same as the `EmptyState`. Why do we have two components with same logic?